.github/workflows/pull-request.yml

```
@@ -5,59 +5,78 @@ on:
       - '**'
 
+
 jobs:
   CodeRabbit-Approval:
     name: Validate CodeRabbit.ai Approval
     runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]'
     steps:
-      - name: Wait for CodeRabbit.ai Approval
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)